internal/cluster.go

```diff
@@ -363,10 +360,19 @@ func (cs *clusterService) GetMemberByUUID(uuid string) core.Member {
 	return nil
 }
 
+func (cs *clusterService) getOwnerConnectionAddress() *protocol.Address {
+	address, ok := cs.ownerConnectionAddress.Load().(*protocol.Address)
```

Comment:
When can this loading return an error? I think it will always return `ok=true`.

Reply:
if ownerConnectionAddress is nil, ok becomes false